cmds/help.js

```diff
@@ -117,20 +119,23 @@ function cmd_help(lang, msg, args, line, wiki) {
 		else msg.reactEmoji('❓');
 	}
 	else if ( msg.isAdmin() && pause[msg.guild.id] ) {
+		var embed = new MessageEmbed().setFooter( 'Help' ).setTimestamp();
 		var cmdlist = lang.get('help.pause') + '\n';
 		cmdlist += formathelp(helplist.pause, msg, lang);
-		msg.sendChannel( cmdlist, {split:{char:'\n🔹',prepend:'🔹',maxLength}}, true );
+		embed.setDescription( cmdlist );
+		msg.replyMsg( '', {embed}, false, false )
```

Comment:
This will break for languages where the `cmdlist` is above the maximal character limit as it is no longer split into multiple messages.